clouddriver-azure/src/main/groovy/com/netflix/spinnaker/clouddriver/azure/client/AzureResourceManagerClient.groovy

```diff
@@ -64,22 +81,42 @@ class AzureResourceManagerClient extends AzureBaseClient {
     deployment
   }
 
-  ResourceGroup createResourceGroup(AzureCredentials creds, String resourceGroupName, String region) {
+  /**
+   *
+   * @param creds
+   * @param resourceGroupName
+   * @param region
+     * @return
+     */
+  ResourceGroup createResourceGroup(String resourceGroupName, String region) {
     try {
-      ResourceGroup resourceGroup = this.getResourceManagementClient(creds).getResourceGroupsOperations().createOrUpdate(resourceGroupName,new ResourceGroup(region)).resourceGroup
+      //Create an instance of the resource group to be passed as the "parameters" for the createOrUpdate method
+      //Set appropriate attributes of instance to define resource group
+      ResourceGroup resourceGroup = new ResourceGroup()
+      resourceGroup.setLocation(region)
+      client.getResourceGroupsOperations().createOrUpdate(resourceGroupName,resourceGroup).body
 
-      resourceGroup
     } catch (e) {
```

Comment:
Probably missing the proper return here.

In the previous SDK createOrUpdate() call is returning a ResourceGroupCreateOrUpdateResult and not a ResourceGroup. Did that method return type changed in the new JSDK?